setup.py

```diff
@@ -24,6 +24,7 @@
     ],
     packages=[
         'dynamallow',
+        'dynamallow/types',
     ],
```

Comment:
This should be python namespaces, i.e. `dynamallow.types`, however I think we should just switch to using `find_packages`.

    packages=find_packages('dynamallow'),

We'll also need to add `find_packages` to the import on line 1.